.github/workflows/scripts/babel_ts_serialize.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright 2018-2020 Gamebridge.ai authors. All rights reserved. MIT license.
+
+import babelCore from "https://dev.jspm.io/@babel/core";
+import presetEnv from "https://dev.jspm.io/@babel/preset-env";
+
+const [, source] = await Deno.bundle("./mod.ts");
+const { code } = babelCore.transformSync(source, {
+  filename: "dist/index.js",
+  presets: [presetEnv],
+  babelrc: false,
+  configFile: false,
+});
+
+console.log(code);
```

Comment:
Just a quick thing to note: This will add a newline to the end of the output produced by babel. If we want to avoid that it's possible to write directly to stdout:
`await Deno.stdout.write(new TextEncoder().encode("Some Text"));`